### DISCUSSION TITLE
Separate pvw and pgm logic, and tally atem aux'es?

### DISCUSSION THREAD

**Discussion by yonazdk** [josephdadams/TallyArbiter#555] [Ideas]
Hi everybody :)

So, I'm using an ATEM Constellation 2M/E as a front end for vMix AND for cutting to IMAG screens.
It's hooked up like this:

M/E 2 -> AUX 1 -> IMAG Screens
M/E 1 -> AUX 9 -> vMix in 1
AUX 10 -> vMix in 2
AUX 11 -> vMix in 3
AUX 12 -> vMix in 4

I would like to tally ATEM input 1-8 ME/2 PGM with no links, with or without PVW.
Then I would like to tally ATEM input 1-8 M/E1 PGM all linked with vMix input 1.
And ATEM input 1-8 M/E1 PVW by itself, and NOT tally vMix input 1 - because if I do, all tallies light up when vMix input 1 is in PVW and something else is in PGM.

Ideally, also tally:
ATEM AUX 10 linked with vMix input 2 (no PVW)
ATEM AUX 11 linked with vMix input 3 (no PVW)
ATEM AUX 12 linked with vMix input 4 (no PVW)

So for this to be possible, I would need a few things:
- Different source selections and link options for PGM and PVW
- Stackable sources or destinations (currently, a clear source will override an active source if more destinations are patched to the same output, and just give a red blink before going black)
- Tally of ATEM aux'es

Any thoughts on this?

**Comment by Jocke4f** [comment]
I have a proposal for Tally AUX support for ATEM AUX in https://github.com/Jocke4f/TallyArbiter/tree/feat_atem_aux. I don't have access to an ATEM with AUX so I can't test it.

**Reply by philannat** [reply]
Thanks a lot Jocke4f for your proposal. I've just found your message and I hope your solution could resolve my problem (aux support for atem). I have an Atem Televison Studio hd. For the moment, Tally Arbiter works great for preview and program. I've downloaded your new file Blackmagicatem.ts and replace the actual file with it (in the directory C:\Users\a\Documents\GitHub\TallyArbiter\src\sources. I've restardted Tally Arbiter, but unfortunatly when I try to use aux output, the m5stickc plus is not changing is color. Do I have to also change the code of the listener ? I use Tally Arbiter 3.0.8. Thanks a lot for your work and your help if you can give me some advice to make it works ... Have a nice day ans may God Bless you a lot. philippe

**Comment by Jocke4f** [comment]
The support for AUX busses from ATEM devices was added in https://github.com/josephdadams/TallyArbiter/pull/669.
